package.json

```
@@ -4,21 +4,24 @@
   "main": "index.js",
   "license": "MIT",
   "engines": {
-    "node": ">=16 < 17"
+    "node": "^16.0.0 || ^18.0.0"
```

```suggestion
    "node": ">=16"
```
maybe this is most flexible